wasm-coverage.md

```diff
@@ -32,6 +34,74 @@ Function Coverage
          <notCoveredFuncs> NCOV => NCOV [ ADDR <- fcd(MOD, ADDR, OID)] </notCoveredFuncs>
       requires notBool ADDR in_keys(NCOV)
       [priority(10)]
+```
+
+Block Coverage
+--------------
+
+```k
+    syntax BlockCoverage ::= blockCoverage ( blockId: Int )
+                           | ifCoverage    ( blockId: Int , truebr: Bool , falsebr: Bool )
+                           | loopCoverage  ( blockId: Int , times: Int )
+                           | "#pause"
+ // --------------------------------------------------------------------
```

Comment:
```suggestion
 // ---------------------------------
```

Reply:
Actually, what does `#pause` do? I don't see it used.

Reply:
used for debgging. removed